### PR TITLE
Dvaered equipment

### PR DESCRIPTION
**Pull request by nenau-again**
(dvaered military ships versions descriptions needs improvement)
* Most ships got only slightly modified, with the military version being always strictly superior to civilian (except for Goddard) one through higher engine limit mass.
* The Phalanx was heavily modified.
* Pirate variants got marginally modified to match modification on Dvaered designs.
* Two new Dvaered launchers were added

### CODE REVIEW

File: dat/scripts/equipopt/templates/dvaered.lua [hunk 1]
```diff
@@ -112,7 +120,7 @@ local function equip_dvaered( p, opt_params )
    -- Set some pilot meta-data
    local mem = p:memory()
    mem.equip = { type="dvaered", level="elite" }
-
+print(params.turret)
```
Comment (bobbens) [inline dat/scripts/equipopt/templates/dvaered.lua:123]: Debug string left in.

**Comment by bobbens** [comment]
Did you forget to commit the Super-Fast Charge launcher?

File: dat/scripts/equipopt/templates/dvaered.lua (hunk 1 above)
Comment (nenau-again) [reply dat/scripts/equipopt/templates/dvaered.lua:123]: sorry...

**Comment by nenau-again** [comment]
I totally did hehe.

**Pull request merged**